python/ray/experimental/test/placement_group_test.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+import ray
+from ray.experimental import PlacementStrategy
+import ray.tests.conftest
+from ray.tests.conftest import ray_start_10_cpus
+
+
+def test_placement_group(ray_start_10_cpus):
+    group = ray.experimental.placement_group("test_group")\
+        .set_strategy(PlacementStrategy.SPREAD)\
+        .add_bundle({"CPU": 2.0}).add_bundle({"CPU": 1.0}).create()
```

Comment:
For python, we probably want these to be kwargs instead of using the builder pattern.

Reply:
`ray.experimental.placement_group(name="test_group", strategy="spread", bundles=[{"CPU": 2}] * 10)`

Reply:
Sounds good.